projects/examples/src/components/datagrid/datagrid-three-renderers.example.component.scss

```diff
@@ -0,0 +1,16 @@
+/*!
```

Comment:
We should not be mixing examples, can we have an example for how to set column width? Can we show some `percentages`, some using `em`, some using `pt`, and probably `px` too.

Right now there's nowhere in the app that would lead me to find this feature.

Reply:
We should embedding values in a selector name. It's bad practice, I know in this case it doesn't matter because there's no real semantic meaning, but it sets a bad example.

`col-a, col-b, col-c`. You can probably show how you can do it without any TS, just using `nth-of-type`.

Make sure the example text narrates what it's trying to show.